conf/modules.config

```diff
@@ -31,22 +31,15 @@ process {
             mode: params.publish_dir_mode
         ]
     }
-    withName: SAVE_REFERENCE {
+    withName: 'GRZQC:PREPARE_REFERENCES:(SAMTOOLS_BGZIP|SAMTOOLS_FAIDX|BWAMEM2_INDEX|MINIMAP2_INDEX)' {
+        ext.prefix = { "${meta.id}" }
         publishDir = [
-            path: params.save_reference_path ? params.save_reference_path : params.outdir,
+            path: { "${params.reference_path}/${meta.id}" },
```

Comment:
meta.id is being reference name here? such as GRCh38 or GRCh37? did you also consider cases where user might use another reference path (then reference name will change)

Reply:
I think it will work